plugins/jobs/web_client/views/JobListWidget.js

```diff
@@ -260,7 +307,33 @@ var JobListWidget = View.extend({
             filter.statuses = JSON.stringify(this.statusFilter);
         }
         this.collection.params = filter;
+
         return this.collection.fetch({}, true);
+    },
+
+    _cancelJobs: function () {
+        Promise.all(
```

Comment:
I have another PR that has the Promise.all like $.whenAll, I am hoping that PR could go in before this one, then I'll merge from master to change this to jQuery Promise.